tests/phpunit/integration/PluginTest.php

```diff
@@ -64,6 +64,10 @@ public function test_register() {
 		$this->assertEquals( 0, did_action( 'googlesitekit_init' ) );
 		do_action( 'init' );
 		$this->assertEquals( 1, did_action( 'googlesitekit_init' ) );
+
+		// Ensure googlesitekit cron events are cleared.
+		$this->assertFalse( wp_get_schedule( 'googlesitekit_cron_daily', array( 'interval' => 'daily' ) ) );
+		$this->assertFalse( wp_get_schedule( 'googlesitekit_cron_hourly', array( 'interval' => 'hourly' ) ) );
```

Comment:
In order to properly test this, I think this test should schedule the events at the beginning before calling `register()`. Otherwise this would always result in true during Travis-CI builds and not test what's needed.